.eslintrc.json

```diff
@@ -1,4 +1,5 @@
 {
+    "parser": "babel-eslint",
```

Comment:
Added this line to eslintrc.json file as I was getting error(unexpected token =) when I used function expression inside a class. Is there other way to remove this error?

Reply:
Hay thats ok :), I think it should be added to the config. I will discuss about it and create a pull request.